src/pybind/tests/test_feat.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import unittest
+import numpy as np
+import os
+import sys
+
+# Add .. to the PYTHONPATH
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))
+import kaldi_pybind as k
+import kaldi_pybind.feat as feat
+from kaldi import SequentialWaveReader
+from kaldi import SequentialMatrixReader
+
+class TestFeat(unittest.TestCase):
+
+    def test_mfcc(self):
+        mfcc = feat.Mfcc(feat.MfccOptions())
+        reader = SequentialWaveReader("ark:tests/wav.ark")
+        # gold set is feature extracted using featbin/compute-mfcc-feats
+        gold_reader = SequentialMatrixReader("ark:tests/feat.ark")
+        for key, value in reader:
+            print("Validate utterance: {}".format(key))
+            self.assertEqual(value.SampFreq(), 16000)
+
+            wave_data = value.Data()
+            nd = np.array(wave_data, copy=False)
+            nsamp = wave_data.NumCols()
+            self.assertAlmostEqual(nsamp, value.Duration() * value.SampFreq(), places = 1)
```

Comment:
did you use an auto formatter for python ? 

`places = 1` should be `places=1` because it is in the argument list.

Reply:
that's a typo.